tradeexecutor/strategy/pandas_trader/position_manager.py

```diff
@@ -392,20 +392,20 @@ def open_1x_long(self,
 
         if stop_loss_pct is not None:
             assert 0 <= stop_loss_pct <= 1, f"stop_loss_pct must be 0..1, got {stop_loss_pct}"
-            position.stop_loss = price_structure.mid_price * stop_loss_pct
+            self.update_stop_loss(price_structure.mid_price * stop_loss_pct, price_structure.mid_price, position)
```

Comment:
Can you make it other way around, position as the first argument. Because all other APIs have it this wa around,